fix(docs): tutorials tags cleanup final batch (#3318)
scaleway/docs-content: tutorials/sinatra/index.mdx

## Creating your first Sinatra Application

1. Create a new file that you can call `testapp.rb`.
    ```
    nano testapp.rbna
    ```
2. Paste the following:
    ```
    require "sinatra"

    get "/" do
      "OMG, a big HELLO to my first app with Sinatra!"
    end
    ```

> 3.  Run your little app. The parameter `-o 0.0.0.0` will bind the application to all interfaces of the server. Without this parameter, it will only be accessible from your localhost.

Reviewer: ```suggestion
3.  Run your Sinatra application. The `-o 0.0.0.0` parameter will bind the application to all interfaces of the server. Without this parameter, it will only be accessible from your localhost.
```

    ```

    The response tells us the port that it is running on. This time it is `4567`.
4. Point the browser to `http://server-IP:4567`.
    <Lightbox src="scaleway-firstapp.webp" alt="" />

## Creating routes

Calls to methods like `get`, `post`, `put`, and `delete` are called routes in Sinatra. They take a path, and a block that handles a request. The request is being taken over by the web server, routed to a piece of code that handles the request and then specifies the response.

> In the example above, The `get '/' do` part of the code is very important. It tells what to do when getting a slash if there is no specified path in the URL after the port.

Reviewer 2: ```suggestion
In the example above, The `get '/' do` part of the code is very important. It says what to do when getting a slash if there is no specified path in the URL after the port.
```

Each item in the `@songs` array is called a _hash_ and is denoted by `{}`.

> In a hash items are stored in key/value pairs, enabling access to a value by its key.

Reviewer: ```suggestion
In a hash, items are stored in key/value pairs, enabling access to a value by its key.
```
